src/main/java/io/pakland/mdas/githubstats/domain/Organization.java

```diff
@@ -32,4 +30,34 @@ public class Organization {
     orphanRemoval = true
   )
   private List<Team> teams = new ArrayList<>();
+
+  public void addTeam(Team team) {
+    if (teams == null) {
```

Comment:
This is true if we are not using the `builder` function. Otherwise, the `teams` would be `null`.

Reply:
I do not understand this part:

> We don't have any teams setter to be afraid (and we shouldn't) to avoid setTeams(null). In the future, if needed, a removeTeam method should be added.

Wasn't this what was previously implemented? I thought it is what you meant with the comment about bidirectional relations. What am I missing?